src/web/mjs/connectors/Cuutruyen.mjs

```diff
@@ -0,0 +1,117 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+import { render_image, init } from './Cuutruyen_wasm.mjs';
+
+export default class Cuutruyen extends Connector {
+    constructor() {
+        super();
+        super.id = 'cuutruyen';
+        super.label = 'Cứu Truyện';
+        this.tags = ['manga', 'vietnamese'];
+        this.url = 'https://cuutruyen.net';
+        this.api = 'https://kakarot.cuutruyen.net';
+
+        this.wasm = init(fetch("https://cuutruyen.net/d660d30a7af3c1ad.module.wasm"));
```

Comment:
I would have some security concerns, loading the `wasm` code directly into HakuNeko's render process which may be used for injection attacks in case the website owners want to attack HakuNeko users.

Reply:
Yeah thats the main problem. If wasm capabilities is just doing calculus (no internet, no file access ? ) the only thing they can do is wasting cpu time. Which is already a problem.

Reply:
> I would have some security concerns, loading the `wasm` code directly into HakuNeko's render process which may be used for injection attacks in case the website owners want to attack HakuNeko users.

I can place the file inside the source tree instead, but I have no idea where I should place it.

Reply:
Sorry about the late reply but i if the internal browser is able to load that WASM shouldnt hakuneko be able to inject a javascript script that calls the WASM function? Like i did in https://github.com/manga-download/hakuneko/pull/6528 ? 

This is how its done in Haruneko so i think it could be ok in Haku.